apps/cyberstorm-remix/.env

```
@@ -4,8 +4,8 @@ SENTRY_PROJECT=
 SENTRY_AUTH_TOKEN=
 ENABLE_BROKEN_PAGES=
 
-VITE_SITE_URL=https://thunderstore.io
-VITE_API_URL=https://thunderstore.io
-VITE_AUTH_BASE_URL=https://auth.thunderstore.io
-VITE_AUTH_RETURN_URL=https://new.thunderstore.io
+VITE_SITE_URL=
+VITE_API_URL=
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The VITE_API_URL key should go before the VITE_SITE_URL key

(UnorderedKey)